### PR TITLE
{vis}[GCCcore/9.3.0] freetype v2.10.1, fontconfig v2.13.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@akesandgren my apologies, I did not mention it in the whislist but `expat-2.2.9` is also in PR #10097

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/da7133a1581a83515c7df6d4fed85ad9 for a full test report.

File: easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.13.1'
```
Comment (ocaisa) [inline easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.1-GCCcore-9.3.0.eb]: There's a version of fontconfig 2.13.92 out there ...

Comment (akesandgren) [reply easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.1-GCCcore-9.3.0.eb]: Doh, totally missed that when I was looking...
Updating....

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/927285fd143a13a2fca5d47fde47147b for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/aaaab84f69629a5c639e3f8c931374b4 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/300b8b1cc5a843b9bce06e0b268697a1 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 5 out of 7 (4 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/205608fa6fe83e8ffbc1d03aa7b27cc2 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (4 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/1440012db539f0a77a39b6dfe3ee7b9d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0f5b82108ca0efedc9bba3bc8dfea024 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/96195dbfbbac8f71acfd012add3460b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2491.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ef003efe0d0756f8e02a6d711e8a3b61 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**